src/compiler/crystal/command/repl.cr

```diff
@@ -34,14 +35,25 @@ class Crystal::Command
     end
 
     if options.empty?
+      splash
```

Comment:
_Splash_ has IMO _splash **screen**_ connotations, I'd suggest using a _banner_ (as in _welcome banner_ - see SSH for instance), or _welcome message_ instead.

Reply:
Sounds good. Now, what do we make of the env var? In the login of Ubuntu you can _hush_ it. But `CRYSTAL_HUSH` sounds a bit rough... `CRYSTAL_NO_MESSAGE`?

Reply:
`CRYSTAL_SKIP_GREETINGS`? ;)

Reply:
I think the env var should mention the interpreter. Maybe `CRYSTAL_INTERPRETER_SKIP_BANNER`?

Reply:
Perhaps _interactive_ could be shorten to `ICR`?

Reply:
Could be, but I prefer `INTERPRETER`